pom.xml

```diff
@@ -90,9 +90,4 @@
     </site>
   </distributionManagement>
 
-  <properties>
-    <jenkins.version>1.580.1</jenkins.version>
```

Comment:
I would keep these properties just to show the example.
Java 7 and 1.625.3 could be the best values (default ones)